profiler/src/ProfilerEngine/Datadog.Profiler.Native/LibddprofExporter.cpp

```diff
@@ -125,6 +148,56 @@ LibddprofExporter::Tags LibddprofExporter::CreateTags(IConfiguration* configurat
     return tags;
 }
 
+std::string LibddprofExporter::GetEnabledProfilersTag(IConfiguration* configuration)
+{
+    const char* separator = "_";  // ',' are not allowed and +/SPACE would be transformed into '_' anyway
+    std::stringstream buffer;
+    bool emptyList = true;
+
+    if (configuration->IsWallTimeProfilingEnabled())
+    {
+        buffer << "walltime";
+        emptyList = false;
+    }
+    if (configuration->IsCpuProfilingEnabled())
+    {
+        if (!emptyList)
+        {
+            buffer << separator;
+        }
+        buffer << "cpu";
+        emptyList = false;
+    }
+    if (configuration->IsExceptionProfilingEnabled())
+    {
+        if (!emptyList)
+        {
+            buffer << separator;
+        }
+        buffer << "exceptions";
+        emptyList = false;
+    }
+    if (configuration->IsAllocationProfilingEnabled())
+    {
+        if (!emptyList)
+        {
+            buffer << separator;
+        }
+        buffer << "allocations";
+        emptyList = false;
+    }
+
+    if (emptyList)
+    {
+        return "";
+    }
+    else
+    {
+        return buffer.str();
+    }
```

Comment:
you can just `return buffer.str()` to simplify, no?

Reply:
✅ done